dvc/command/add.py

```diff
@@ -74,6 +77,24 @@ def add_parser(subparsers, parent_parser):
         help="Specify name of the DVC-file this command will generate.",
         metavar="<filename>",
     )
+    parser.add_argument(
+        "-o",
+        "--out",
+        help="Destination path to put files to.",
+        metavar="<filename>",
+    )
+    parser.add_argument(
+        "--to-remote",
+        action="store_true",
+        default=False,
+        help="Download it directly to the remote",
```

Comment:
How about something like
```suggestion
        help="Implies `--external`. Don't put `targets` in the workspace "
              "nor cache them, but push them to remote storage.",
```

Reply:
But now that I see this I realize it's indeed a little unexpected that `add` includes `push`...